xml/System.Net.Security/SslStreamCertificateContext.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Represents a set of certificates used for building a certificate chain.</summary>
```

Comment:
I would add note that the certificates are normalized and if something is missing in constructed X509Chain, SslStream will try to resolve them - either by searching certificates stores or by fetching them via network if allowed.